Notification/template.yaml

```diff
@@ -26,13 +90,108 @@ Resources:
           Properties:
             Path: /hello
             Method: get
+  
+  UserNotificationsFunction:
```

Comment:
Functions to:
1. POST /v1/users/{userId}/notifications/{fridgeId}: user adds a new notification preferences for a specified fridge
2. PUT /v1/users/{userId}/notifications/{fridgeId}: user edits a notification preference for a specified fridge
3. GET /v1/users/{userId}/notifications/{fridgeId}: user gets notification preferences for a specified fridge
4. GET /v1/users/{userId}/notifications: user gets all the notification preferences for specified user